src/minio_governance/models.py

```diff
@@ -0,0 +1,122 @@
+"""
+Pydantic models for CDM MinIO Data Governance API responses
+"""
+
+from datetime import datetime
+from typing import Annotated, Any, Dict, List, Optional
+
+from pydantic import BaseModel
+
+
+class CredentialsResponse(BaseModel):
+    """Response model for user credentials"""
+
+    username: Annotated[str, "Username"]
+    access_key: Annotated[str, "MinIO access key (same as username)"]
+    secret_key: Annotated[str, "MinIO secret key (fresh on each request)"]
```

Comment:
How does this documentation show up in the notebook?

Reply:
hmmm I am not sure I am following you. It would be something like a dict.

Reply:
This is moot since these classes aren't intended for user consumption if I understand you correctly